### DISCUSSION TITLE
Quarto publishing a website on Windows

### DISCUSSION THREAD

**Discussion by kjayhan** [quarto-dev/quarto-cli#4589] [Q&A] [answered]
I am able to publish updates to my website on a Mac, but not on Windows.

I remember reading that Quarto was still working on making it possible (or easier) for publishing Quarto websites on Windows. 

I don't know if it is still an issue for all Windows, but when I tried quarto publish gh-pages on Rstudio's terminal on Windows, it gives me only two options: Netlify and Rstudio Connect, and not Github Pages where my website is hosted.

Any ideas?

**Comment by dragonstyle** [comment]
Could you try a newer version of Quarto (perhaps even our pre-release version 1.3 https://quarto.org/docs/download/prerelease)? I believe this is something that we had walled off while we fixed some pretty severe issues and we've opened it back up now that those issues have been resolved.

**Reply by kjayhan** [reply, marked as answer]
> Could you try a newer version of Quarto (perhaps even our pre-release version 1.3 https://quarto.org/docs/download/prerelease)? I believe this is something that we had walled off while we fixed some pretty severe issues and we've opened it back up now that those issues have been resolved.

I can confirm now that Quarto websites can be published on Windows as well. I have done so today with the following session info:

R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)
